Library/Util.ts

```diff
@@ -151,5 +154,29 @@ class Util {
 
         return map;
     }
+
+    /**
+     * Checks if a request url is not on a excluded domain list 
+     * and if it is safe to add correlation headers (x-ms-request-source-ikey, x-ms-request-target-ikey)
+     */
+    public static canIncludeCorrelationHeader(client: Client, options: http.RequestOptions | http.ServerResponse) {
+        let excludedDomains = client && client.config && client.config.correlationHeaderExcludedDomains;
+        if (!excludedDomains || excludedDomains.length == 0 || !options) {
+            return true;
+        }
+
+        let url = (<http.RequestOptions>options).host ||
+                ((<http.ServerResponse>options).getHeader && (<http.ServerResponse>options).getHeader("host"));
```

Comment:
It's to disable the `x-ms-request-target-ikey` header - [see](https://github.com/Microsoft/ApplicationInsights-node.js/pull/156/files#diff-641655496f5e68858e3a1056f5072f06). I was wondering if I should only disabled `x-ms-request-source-ikey` header, but I've decided to disable all correlation headers for excluded domains.